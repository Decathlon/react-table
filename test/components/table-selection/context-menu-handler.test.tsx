/// <reference path="../../typings/tests-entry.d.ts" />
import * as React from "react";
import { createRenderer } from "react-test-renderer/shallow";

import ContextMenuHandler from "../../../src/components/table-selection/context-menu-handler";

describe("ContextMenuHandler component", () => {
  test("should render children", () => {
    const children = () => <div>Foo</div>;
    const props = {
      children,
      selectedCells: {}
    };
    const shallowRenderer = createRenderer();
    shallowRenderer.render(<ContextMenuHandler {...props} />);
    const rendered = shallowRenderer.getRenderOutput();
    expect(rendered).toMatchSnapshot();
  });
});
