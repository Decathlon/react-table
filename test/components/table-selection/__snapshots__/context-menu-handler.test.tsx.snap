// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContextMenuHandler component should render children 1`] = `
<React.Fragment>
  <div>
    Foo
  </div>
</React.Fragment>
`;
